### PR TITLE
Upgrade Ballerina version to 2201.5.0

### PR DESCRIPTION
**Pull request by Thiyan**

### CODE REVIEW

File: base/Ballerina.toml [hunk 1]
```diff
@@ -2,7 +2,7 @@
 org = "ballerinax"
 name = "health.base"
 version = "1.0.2-snapshot"
```
Comment (ThishaniLucas) [inline base/Ballerina.toml]: we need to bump the version as well for the release.

Comment (Thiyan) [reply base/Ballerina.toml]: This should be done in the  Prepare for release commit.. right?

Comment (Thiyan) [reply base/Ballerina.toml]: Updated

**Review by AnjanaSenanayake — APPROVED** [review]

File: base/Dependencies.toml [hunk 2]
```diff
@@ -322,7 +325,7 @@ modules = [
 [[package]]
 org = "ballerinax"
 name = "health.base"
```
Comment (AnjanaSenanayake) [inline base/Dependencies.toml:327]: This is `ballerinax/health.base` package right. Why do we have the package itself is a dependency?

Comment (Thiyan) [reply base/Dependencies.toml:327]: It is auto-generated. expected behaviour right?

**Pull request merged**